pyinstxtractor.py

```diff
@@ -238,14 +240,19 @@ def _writeRawData(self, filepath, data):
             f.write(data)
 
 
-    def extractFiles(self):
+    def extractFiles(self, custom_dir=None):
```

Comment:
This could have been done like below to reduce the code duplication

```python
    def extractFiles(self, extractionDir=None):
        print('[+] Beginning extraction...please standby')
        if extractionDir is None:
                extractionDir = os.path.join(os.getcwd(), os.path.basename(self.filePath) + '_extracted')

        if not os.path.exists(extractionDir):
            os.mkdirs(extractionDir)
```

Reply:
Correspondingly, `sys.argv[2]` can stand for the optional extraction directory. Necessary changes need to be made to `main`.